src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -31,39 +57,44 @@ public String todayDate(DateTimePart datePart) {
      *                   - 3-й элемент массива - день (число);
      */
     public Optional<LocalDate> getDate(Integer[] dateParams) {
-        return Optional.empty();
+        try {
+            LocalDate date = LocalDate.of(dateParams[0], dateParams[1], dateParams[2]);
+            return Optional.of(date);
+        } catch (DateTimeException | ArrayIndexOutOfBoundsException e) {
+            return Optional.empty();
+        }
```

Comment:
```suggestion
        } catch (DateTimeException | ArrayIndexOutOfBoundsException e) {
            System.out.println(e.getMessage()); // this should be logged, but for now we can sout this
        }
        return Optional.empty();
```
in this particular case - when we catch two different exceptions, we should be able to identify them somehow